<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="serialCb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openBt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>71</y>
      <width>296</width>
      <height>482</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>驻留方位角度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fwjdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="fwjdBt">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>驻留俯仰角度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zlfyjdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="zlfyjdBt">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>圆扫方位速度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ysfwsdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="ysfwsdBt">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>圆扫方位角度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ysfwjdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="ysfwjdBt">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>扇扫方位速度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ssfwsdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="ssfwsdBt">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>扇扫俯仰速度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ssfysdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="ssfysdBt">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>手动方位角度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sdfwjdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="sdfwjdBt">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="0">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="14" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>手动俯仰角度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sdfyjdEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="sdfyjdBt">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
